projects/step-core/src/lib/components/link-button/link-button.component.ts

```diff
@@ -28,6 +28,6 @@ export abstract class LinkButtonComponent implements OnInit {
 
     from(this._router.navigateByUrl('/'))
       .pipe(switchMap(() => timer(100)))
-      .subscribe(() => this._router.navigateByUrl(this.url!));
+      .subscribe(() => this._router.navigateByUrl(this.url!, { replaceUrl: true }));
```

Comment:
what's the purpose of this replaceUrl? also, do we need the timer() thing?

Reply:
@lucianbaciu 
I've rechecked it and removed the timer. It's unnecessary now.
It was added to handle some issues when both angular & angularJS routing were used on the settings screen.
Now as this part it migrated, timer not needed anymore here.

`replaceUrl` parameter overrides last location change. So during the back navigation `this._router.navigateByUrl('/')` will be ignored. Nearly the same operation like `skipLocationChange`